httplib.h

```diff
@@ -2960,8 +2977,11 @@ class MultipartFormDataParser {
           auto header = buf_.substr(0, pos);
           {
             std::smatch m;
-            if (std::regex_match(header, m, re_content_type)) {
-              file_.content_type = m[1];
+            const std::string header_name = "content-type:";
+            if (contains_header(header, header_name)) {
+              header.erase(header.begin(), header.begin() + 13);
```

Comment:
Maybe replace magic number 13 with header_name.size()?
```suggestion
              header.erase(header.begin(), header.begin() + header_name.size());
```